tools/2d_feature_extraction/2d_feature_extraction.py

```diff
@@ -0,0 +1,126 @@
+#!/usr/bin/python
+
+import argparse
+import numpy as np
+import pandas as pd
+import tifffile
+import skimage.io
+import skimage.measure
+import skimage.feature
+import skimage.segmentation
+import skimage.morphology
+
+#TODO make importable by python script
+
+parser = argparse.ArgumentParser(description='Extract Features 2D')
+
+#TODO create factory for boilerplate code
+features = parser.add_argument_group('compute features')
```

Comment:
Btw. do you know about https://github.com/erasche/argparse2tool

It gives you at least a very rough template tool and might save you some time.